ccd-definition/ComplexTypes/DocumentOrImageWithRegex.json

```diff
@@ -5,6 +5,6 @@
     "FieldType": "Document",
     "ElementLabel": "Document",
     "SecurityClassification": "Public",
-    "RegularExpression": ".pdf,.txt,.doc,.dot,.docx,.rtf,.xls,.xlt,.xla,.xlsx,.xltx,.xlsb,.ppt,.pot,.pps,.ppa,.pptx,.potx,.ppsx,.jpg,.jpeg,.bmp,.tif,.tiff,.png"
+    "_RegularExpression": ".pdf,.txt,.doc,.dot,.docx,.rtf,.xls,.xlt,.xla,.xlsx,.xltx,.xlsb,.ppt,.pot,.pps,.ppa,.pptx,.potx,.ppsx,.jpg,.jpeg,.bmp,.tif,.tiff,.png"
   }
 ]
```

Comment:
Waiting for https://tools.hmcts.net/jira/browse/CIV-7265 to fix regex issue, but will not be merged as this regardless

Reply:
Not sure I get it.

Will this particular change be merged as part of this PR or not?